pinc/Project.inc

```diff
@@ -768,14 +769,30 @@ function does_project_page_table_exist($projectid)
 // long form (projectID####) or just the project number (####).
 // If it does, it returns it, otherwise dies with a warning.
 function validate_projectID($param_name, $value, $allownull = false)
+{
+    try
```

Comment:
You don't need the try/catch block at all, just make the call to the file that will throw the exception. We have an exception handler that will output the error message. See `pinc/base.inc`